### PR TITLE
Sorting out some css inconsistencies of tables inside termynal elements

### PR DESCRIPTION
**Pull request by nhussein11**
This PR addresses and resolves CSS inconsistencies related to tables within Termynal elements

### CODE REVIEW

**Comment by nhussein11** [comment]
@eshaben Not sure if I like these borders; could you please take a look at them?

File: material-overrides/assets/stylesheets/terminal.css [hunk 1]
```diff
@@ -66,8 +66,19 @@
...
     background-color: unset;
-    border-bottom: .05rem solid var(--md-typeset-table-color);
+    border-top: .05rem solid var(--md-typeset-table-color);
     border-right: .05rem solid var(--md-typeset-table-color);
+    color: var(--md-typeset-table-color);
+}
+
+.md-typeset .table-termynal .md-typeset__table table td {
+    color: var(--md-typeset-table-color);
 }
```
Comment (eshaben) [inline material-overrides/assets/stylesheets/terminal.css:84]: What if you try this:

```
.md-typeset .table-termynal table:not([class]) {
    border: none;
}

.md-typeset .table-termynal .md-typeset__table table th.left-header {
    border-bottom: .05rem solid rgba(255,255,255,.1);
    border-right: .05rem solid rgba(255,255,255,.1);
    border-color: rgba(255,255,255,.1);
    background-color: var(--black);
}

.md-typeset .table-termynal .md-typeset__table table th.center-header {
    text-align: center;
    width: 100%;
    color: var(--white);
    background-color: var(--pink);
    padding: 0 22% 0 0;
}

.md-typeset .table-termynal table:not([class]) td {
    background-color: var(--black);
    border-color: rgba(255,255,255,.1);
}
```

Comment (eshaben) [reply material-overrides/assets/stylesheets/terminal.css:84]: It makes the table the same colors on light and dark mode (which in a terminal element, it probably should be this way).

I personally like the more subtle borders, but we can change it up - whatever works

File: material-overrides/assets/stylesheets/polkadot.css [hunk 2]
```diff
@@ -80,6 +80,8 @@ body {
   --md-code-hl-keyword-color: var(--blue);
   /* Comments */
   --md-code-hl-comment-color: #545454;
+  /* Table content */
+  --md-typeset-table-color: #545454;
```
[the remark this answers is not in the document]
Comment (nhussein11) [reply material-overrides/assets/stylesheets/polkadot.css]: These colors are quite opaque IMO, or maybe it's just my monitor:
image

I don't like them tbh hahaha, but I'm not against it just fyi @eshaben

File: material-overrides/assets/stylesheets/terminal.css (hunk 1 above)
Comment (nhussein11) [reply material-overrides/assets/stylesheets/terminal.css:84]: This does the trick 100%, LGTM!

**Pull request merged**